docbits/54_shobject.md

```diff
@@ -2,79 +2,83 @@
 
 In addition to the base sheet keys, worksheets also add:
 
-- `ws['!cols']`: array of column properties objects.  Column widths are actually
+- `ws['!cols']`: array of column properties objects. Column widths are actually
   stored in files in a normalized manner, measured in terms of the "Maximum
-  Digit Width" (the largest width of the rendered digits 0-9, in pixels).  When
+  Digit Width" (the largest width of the rendered digits 0-9, in pixels). When
   parsed, the column objects store the pixel width in the `wpx` field, character
   width in the `wch` field, and the maximum digit width in the `MDW` field.
 
 - `ws['!rows']`: array of row properties objects as explained later in the docs.
   Each row object encodes properties including row height and visibility.
+  If there exists a row which should be hidden pass the `cellStyles:true` option
+  to the read function`XLSX.read(data, {cellStyles:true})` where you attempt
```

Comment:
While your IDE strips double spaces(that's why there are so many changes and it's hard to review, I guess) you missed that one:
```diff
---to the read function`XLSX.read(data, {cellStyles:true})` where you attempt
+++to the read function `XLSX.read(data, {cellStyles:true})` where you attempt
```